src/parseExpensiMark.ts

```diff
@@ -273,30 +254,24 @@ function groupRanges(ranges: MarkdownRange[]) {
   }, [] as MarkdownRange[]);
 }
 
-function parseExpensiMarkToRanges(markdown: string): MarkdownRange[] {
+function parseExpensiMark(markdown: string): MarkdownRange[] {
   if (markdown.length > MAX_PARSABLE_LENGTH) {
     return [];
   }
-  try {
```

Comment:
It was a deliberate decision to remove try/catch block so we can show a RedBox with the error if something fails.

However, looks like I forgot about the case when Markdown message before and after parsing don't match. I think we should show a warning and return an empty array in such case, like this:

```tsx
if (text !== markdown) {
  console.warn(
    `[react-native-live-markdown] Parsing error: the processed text does not match the original Markdown input. This may be caused by incorrect parsing functions or invalid input Markdown.\nProcessed input: '${JSON.stringify(
      text,
    )}'\nOriginal input: '${JSON.stringify(markdown)}'`,
  );
  return [];
}
```